src/components/container/FeaturedJobs/FeaturedJobs.scss

```diff
@@ -49,10 +49,37 @@
     }
   }
 
+  &__viewMoreLink__bottom {
+    display: none;
+  }
+
   &__cardContainer {
     display: flex;
     gap: 32px;
     justify-content: space-between;
     flex-wrap: wrap;
   }
+
+  @media screen and (max-width: 840px) {
+    &__viewMoreLink {
+      display: none;
+    }
+
+    &__viewMoreLink__bottom {
```

Comment:
I would use a modifier, then you could also omit the duplicate specifications for the icon. But that's not merge blocking, so still approving :)